lib/src/chip/chip.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'hc-chip',
+    templateUrl: './chip.component.html',
+    styleUrls: ['./chip.component.scss']
+})
+export class ChipComponent implements OnInit {
+
+    @Input() action: boolean = false;
+    @Input() color: string = 'neutral';
+    chipCursor: string = 'auto';
+    chipDisplay: string = 'inline-block';
+    chipType: string = 'chip';
+
+    constructor() {
+    }
+
+    ngOnInit() {
+        if (this.action) {
+            this.chipCursor = 'pointer';
+            this.chipType += ' close';
```

Comment:
This action could change so you'll want to move this logic so that it gets re-evaluated. Otherwise, win the user changes the action variable nothing will happen